tests/monotouch-test/CoreGraphics/PDFContextTest.cs

```diff
@@ -76,6 +76,7 @@ public void Context_Url_Rect_Info ()
 		}
 
 		[Test]
+		[Ignore ("Crash on simulator using xcode 9.3 beta 1")]
```

Comment:
This will ignore the test on all iOS versions, IMHO it would be better to have a version + simulator check and only ignore in that case.

Reply:
it was meant as temporary - but that might be wishful thinking on my part